### DISCUSSION TITLE
How to render materail react table in dark theme?

### DISCUSSION THREAD

**Discussion by mohan-subbu** [KevinVandy/material-react-table#973] [General]
Hi Team
I am unable to find a solution on how to render Material react table in dark theme(black)
Currently I am using  "material-react-table": "^1.11.4", and it is rendered in blue and white
<img width="951" alt="image" src="https://github.com/KevinVandy/material-react-table/assets/30363951/993b90b8-03c9-4c8d-ac4e-a6fed853ce9d">

How can i render it in dark theme like below screenshot
<img width="861" alt="image" src="https://github.com/KevinVandy/material-react-table/assets/30363951/2da392b7-c071-4f4b-a60b-5068afd54abd">

Is there any attributes that we can give to table? Can u guys please help me
TIA

**Comment by KevinVandy** [comment]
Material React Table simply follows the MUI theme that you set up.

**Reply by mohan-subbu** [reply]
I am not using MUI or MUI theme anywhere in my application. I am just using Materail react table in one page.
Is there any attributes that we can provide dark theme inputs for it?

**Reply by KevinVandy** [reply]
Make a MUI Theme and wrap it around MaterialReactTable. This is in the [Style Guide](https://www.material-react-table.com/docs/guides/customize-components#customize-theme-just-for-your-table)

**Reply by oleksii-lukin** [reply]
It seems to be broken right now https://github.com/KevinVandy/material-react-table/issues/1429

**Reply by oleksii-lukin** [reply]
Probably since this one https://github.com/KevinVandy/material-react-table/issues/1382
